src/app/shared/tabs/entity-datatable-tab/datatable-multi-row/datatable-multi-row.component.scss

```
@@ -62,6 +62,15 @@
       overflow-wrap: anywhere;
     }
 
+    .mobile-cell-label {
+      display: none;
+    }
+
+    .cell-value {
+      overflow-wrap: anywhere;
+      word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)